_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -146,7 +154,9 @@ proc cmake::handle_generator {option action args} {
             "*Unix Makefiles*" {
                 ui_debug "Selecting the 'Unix Makefiles' generator ($args)"
                 depends_build-delete \
-                                port:ninja
+                                path:bin/ninja:ninja
+                depends_skip_archcheck-delete \
+                                ninja
```

Comment:
The ninja Portfile already specifies `installs_libs no` so using `depends_skip_archcheck` for ninja here should not be necessary.